changelog/2757.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed urllib3.connection.HTTPConnection to raise the http.client.connect audit event to have the same behavior as the standard library HTTP client
```

Comment:
Can you please add single backticks around urllib3.connection.HTTPConnection and http.client.connect? I think it should make links to the relevant documentation

Reply:
Sorry I meant something like this as I had missed that `http.client.connect` was not an Intersphinx target:

```reStructuredText
Fixed :class:`~urllib3.connection.HTTPConnection` to raise the ``http.client.connect`` audit event to have the same behavior as the standard library HTTP client (`#2757 <https://github.com/urllib3/urllib3/issues/2757>`__)
```

But the version that got merged is fine too.